crescita-personale/tirocinio.md

```diff
@@ -0,0 +1,148 @@
+# Tirocinio
+
+In Nebulab, crediamo fermamente nella capacità delle persone di esprimere il proprio potenziale
+quando si trovano nelle condizioni adatte. Per questo abbiamo inserito nel nostro team una posizione
+di tirocinio permanente.
+
+L'obiettivo del tirocinio è quello di inserire sviluppatori alle prime armi nel contesto in cui ci 
+lavoriamo tutti i giorni, col fine di insegnargli le best practice del settore dal punto di vista
+tecnico e delle _soft skill_. Con noi imparerai tutto quello che ti serve per lavorare su progetti
+reali con clienti reali e tutte le responsabilità che ne conseguono.
+
+Il nostro tirocinio dura quattro mesi. Al termine del tirocinio, il tirocinante viene solitamente 
+assunto in Nebulab. Se questo non dovesse essere possibile, lo aiuteremo a trovare una posizione più 
+adatta.
+
+## Selezione
+
+Il processo di selezione è piuttosto semplice e lineare.
+
+### Requisiti base
+
+Innanzitutto, assicurati di possedere i requisiti base per fare domanda, che sono:
+
+- Ottima padronanza dell'italiano (perché è la lingua che parliamo tra noi) e dell'inglese (perché i 
+nostri clienti sono tutti basati negli US)
+- Buona conoscenza dei concetti di base della programmazione (flow control, strutture iterative più 
+comuni) e della programmazione OOP (classi, oggetti ed ereditarietà)
+- Un atteggiamento proattivo al lavoro e all'apprendimento (= non aspettare che ti diciamo cosa fare
+o cosa leggere, ma chiedicelo/trovalo!)
+
+Ci rendiamo conto che questi requisiti potrebbero escludere dei candidati che non hanno esperienza
+con la programmazione ma vorrebbero imparare. Purtroppo, in questo momento insegnare a qualcuno la
+programmazione sarebbe un investimento di tempo troppo grande per noi. Se non hai esperienza con la
+programmazione ma vorresti imparare, ti consigliamo di consultare queste risorse:
+
```

Comment:
Manca la lista ma immagino sia ancora da stabilire cosa metterci qui. Avevi idee in particolare? Tipo linkare al sito dell'università :trollface: oppure risorse online o bootcamp?